e2e/core-e2e/tests/nx-dotnet.spec.ts

```diff
@@ -67,19 +67,14 @@ describe('nx-dotnet e2e', () => {
   });
 
   it('should work with affected', async () => {
+    const ngApp = uniq('ng-app');
     const testApp = uniq('app');
     const testLib = uniq('lib');
 
     runCommand('git checkout -b "affected-tests"', {});
-    updateFile('package.json', (f) => {
-      const json = JSON.parse(f);
-      json.dependencies['@nrwl/angular'] = json.devDependencies['nx'];
-      return JSON.stringify(json);
-    });
-    runPackageManagerInstall();
 
     await runNxCommandAsync(
-      `generate @nrwl/angular:app ng-app --style css --routing false --no-interactive`,
+      `generate @nx/angular:app ${ngApp} --style css --routing false --no-interactive`,
```

Comment:
Maybe I was wrong in removing the fix above, but it made the tests much faster by not requiring reinstalling dependencies.